<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_information</class>
 <widget class="QDialog" name="customer_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 255, 233);</string>
  </property>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 13pt &quot;Bodoni MT Black&quot;;
color:rgb(127, 127, 95);</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="phonenum">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 13pt &quot;Bodoni MT Black&quot;;
color:rgb(127, 127, 95);</string>
   </property>
   <property name="text">
    <string>PhoneNum</string>
   </property>
  </widget>
  <widget class="QLabel" name="require">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 13pt &quot;Bodoni MT Black&quot;;
color:rgb(127, 127, 95);</string>
   </property>
   <property name="text">
    <string>Requirement</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 13pt &quot;Bodoni MT Black&quot;;
color:rgb(127, 127, 95);</string>
   </property>
   <property name="text">
    <string>Pickup Date</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="size">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 13pt &quot;Bodoni MT Black&quot;;
color:rgb(127, 127, 95);</string>
   </property>
   <property name="text">
    <string>Cake Size</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Bodoni MT Black&quot;;
color:rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Bodoni MT Black&quot;;
color:rgb(127, 127, 95);</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="size_choice">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Bodoni MT Black&quot;;
color:rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rb1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1호</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2호</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3호</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 170, 255);
font: 87 30pt &quot;Bodoni MT Black&quot;;</string>
   </property>
   <property name="text">
    <string>Customer Information</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="phone_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="require_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
